evennia/web/website/templates/website/_menu.html

```diff
@@ -40,8 +40,21 @@
             {% endblock %}
             {% block navbar_user %}
             {% if user.is_authenticated %}
-            <li class="nav-item">
-                <a class="nav-link">Logged in as {{user.username}}</a>
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="user_options" aria-expanded="false">Logged in as {{user.username}} <span class="caret"></span></a>
+              <div class="dropdown-menu" aria-labelledby="user_options">
+                <a class="dropdown-item" href="{% url 'chargen' %}">Create</a>
+                <a class="dropdown-item" href="#">Manage</a>
```

Comment:
I think this should be removed - it doesn't do anything and that feels like a bug to user. If it's meant as an example it should show some sort of information saying that it's up to the dev to expand on it.

Reply:
That was my own placeholder; I'm creating a character management menu to go along with this as well but enabling and implementing it hinges on how #1674 gets resolved. For development I just needed Django to not yell at me about nonexistent links :)

Reply:
This now lets users select a puppet to use within the context of the website.